### PR TITLE
Show open/incomplete/closed data by territory

### LINKED ISSUE #1: Show open/incomplete/closed data by territory
When I visit transport.beta.gouv.fr,
    I want to know what territories have opened their data,
    so that I can pick one territory to reuse their data.

    When I visit transport.beta.gouv.fr,
    I want to know what territories haven't yet opened their data,
    so that I push them to do so.

### PR DESCRIPTION
**Pull request by maukoquiroga**
Fixes #1 

Depends on #13, #14, #15, #16 and #17.

Rebase before merge.

capture d ecran 2017-09-09 a 18 04 49

There's styling, JS/Elm porting and other stuff that'll come gradually with other PRs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,7 +17,11 @@ erl_crash.dump
 
 # Since we are building assets from client,
 # we ignore priv/static.
-/priv/static/
+/priv/static/js
+/priv/static/css
+
+# For now, we'll be ignoring data files.
+/priv/static/data/aots
```
Comment (l-vincent-l) [inline .gitignore]: can't you do something like /priv/static/* ?

File: client/webpack.config.js [hunk 2]
```diff
@@ -10,19 +10,28 @@ module.exports = {
...
+            './images/marker-icon-2x.png$': resolve('./node_modules/leaflet/dist/images/marker-icon-2x.png'),
+            './images/marker-shadow.png$': resolve('./node_modules/leaflet/dist/images/marker-shadow.png')
+        },
+        extensions: ['.elm', '.js', '.scss', '.png', '.svg']
     },
     plugins: [extractSass],
     devtool: 'source-map',
     module: {
-        noParse: /\.elm$/,
+        noParse: /\.elm$/i,
```
Comment (l-vincent-l) [inline client/webpack.config.js]: you to be able to capture .ElM ?

Comment (maukoquiroga) [reply client/webpack.config.js]: Done

**Comment by maukoquiroga** [comment]
@l-vincent-l ping.

**Pull request merged**